docs/USAGE.md

```diff
@@ -121,19 +121,11 @@ Make sure to follow the following step.
 With `bitcoind` running, do:
 
 ```
-bitcoin-cli -named createwallet wallet_name=jm_wallet descriptors=false
+bitcoin-cli createwallet jm_wallet
```

Comment:
Need a wallet without private keys, because we're adding descriptors without private keys:
```sh
bitcoin-cli -named createwallet wallet_name=jm_wallet disable_private_keys=true
```
otherwise it spits out the following error:
```
2025-04-22 13:47:55,670 [WARNING]  Cannot import descriptor without private keys to a wallet with private keys enabled
```